sda-commons-server-openapi/README.md

```diff
@@ -0,0 +1,232 @@
+# SDA Commons OpenApi
```

Comment:
I think in this file are still some references to Swagger 2 instead of OpenApi 3.

Reply:
Should be fixed.